src/guiMainMenu.h

```diff
@@ -25,17 +25,19 @@ with this program; if not, write to the Free Software Foundation, Inc.,
 #include <string>
 #include <list>
 
-enum
-{
-	TAB_SINGLEPLAYER=0,
-	TAB_MULTIPLAYER,
-	TAB_ADVANCED,
-	TAB_SETTINGS,
-	TAB_CREDITS
+struct MainMenuDataForLua {
+	// Whether the server has requested a reconnect
+	bool reconnect_requested;
+
+	//error handling
+	std::string errormessage;
+
+	MainMenuDataForLua():
```

Comment:
There should be a space between the closing parenthesis and the initializer list colon.